lighthouse-core/audits/deprecations.js

```diff
@@ -59,6 +59,19 @@ class Deprecations extends Audit {
           }, log.entry);
         });
 
+    function makeDepractionsV2(entries) {
```

Comment:
jsdoc would be nice here just to not require looking through the impl to see the return type